internal/app/server/open_saves_test.go

```diff
@@ -99,6 +112,31 @@ func TestOpenSaves_HealthCheck(t *testing.T) {
 	if want := healthgrpc.HealthCheckResponse_SERVING; got.Status != want {
 		t.Errorf("hc.Check got: %v, want: %v", got.Status, want)
 	}
+	cancel()
+}
+
+func TestOpenSaves_RunServer(t *testing.T) {
+	serviceConfig, err := getServiceConfig()
```

Comment:
`t.Parallel()`?

Reply:
I don't want to run these tests in parallel since they use the same port. The timeouts are short enough that there shouldn't be much delay here anyway. Same response for the remaining tests.

Reply:
oh yeah that makes sense, thanks